src/js/core/widget/core/Slider.js

```diff
@@ -715,88 +723,138 @@
 				self._previousValue = self.element.value;
 			};
 
+			/**
+			 * touchstart event handler
+			 * @method _onTouchStart
+			 * @member ns.widget.core.Slider
+			 * @protected
+			 */
 			prototype._onTouchStart = function () {
 				this._ui.handlerElement.classList.add(classes.SLIDER_HANDLER_ACTIVE);
 			};
 
+			/**
+			 * touchend event handler
+			 * @method _onTouchEnd
+			 * @member ns.widget.core.Slider
+			 * @protected
+			 */
 			prototype._onTouchEnd = function () {
 				this._ui.handlerElement.classList.remove(classes.SLIDER_HANDLER_ACTIVE);
 			};
 
-			// prototype._onFocus = function () {
-			// 	var container = this._ui.barElement.parentElement;
-
-			// 	container && container.classList.add("ui-listview-item-focus");
-			// };
-
-			// prototype._onBlur = function () {
-			// 	var container = this._ui.barElement.parentElement;
-
-			// 	container && container.classList.remove("ui-listview-item-focus");
-			// };
-
+			/**
+			 * Decrease slider current value by one step (triggers only input event)
+			 * @method _decreaseValue
+			 * @member ns.widget.core.Slider
+			 * @protected
+			 */
 			prototype._decreaseValue = function () {
 				var self = this;
 
 				self._setValue(self._value - (parseFloat(self.element.step) || 1));
+				events.trigger(self.element, "input");
```

Comment:
I'm afraid that this event may cause looping in app eg. smart-things. This should be checked.